Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
Actually I got some permission deny error (it may need root?) when "make testrace" on my desktop after adding that line. This will impact our users when they make locally. Can you take a look?